### ISSUE TITLE
Why could a multiclass dice loss function solve the class imbalance problem? 

### ISSUE THREAD

**Issue by yianzhongguo** [MIC-DKFZ/BraTS2017#29]
@FabianIsensee Hello, sir. I notice that in the paper of this method the class imbalacnce issue was disposed by formulating a multiclass dice loss function. I cannot figure it out why the loss function could play the role. Could you detailed it, thank you!

**Comment by FabianIsensee** [comment]
In crossentropy, each pixel has the same weight irrespective of the class. by using a Dice loss, the weight of a pixel is different. If the CE tumor is small for example, then false positives or false negatives will impact the dice loss more and will thus intrinsically be weighted more.

**Comment by yianzhongguo** [comment]
I have understood it from your answer, thank you very much!,
